packages/js/src/ui/components/primitives/Tooltip/TooltipContent.tsx

```diff
@@ -28,7 +28,7 @@ const TooltipContentBody = (props: TooltipContentProps) => {
     <div
       ref={setFloating}
       class={local.class ? local.class : style(local.appearanceKey || 'tooltipContent', tooltipContentVariants())}
-      style={floatingStyles()}
+      style={{ ...floatingStyles(), 'z-index': 99999 }}
```

Comment:
`10` value works here, but not sure why it's `10` 😅

Reply:
I don't think `10` would work in a custom popover scenario. You can't know what the host app has rendered and with what `z-index` and since the tooltip is portaled, it might show under something else. Does this make sense?